scripts/components/opencv.sh

```diff
@@ -70,8 +70,6 @@ else
 
 fi
 
-
-
-
-
-
+if [[ $TARGET_ARCHITECTURE == "host" ]]; then
+  cp -R $BUILD_DIR/share/licenses/opencv4 $LICENSE_DIR
+fi
```

Comment:
What about non-host builds? Is the license in `$BUILD_DIR` dependent on which components actually are built or why not use something from `$SRC_DIR`?

Reply:
The OpenCV build is a bit messy here. It's built multiple times for each Android ABI even though only one build would be needed. This is kind of handling that